### ISSUE TITLE
STT_309_Whistler player can return ore to player accidentally in dialogue DIA_Whistler_RunningPayBack.

### ISSUE THREAD

**Issue by Fawkes-dev** [Fawkes-dev/G1-CZ-Revision#72]
This happened to me several times 😊 As I was going through dialogues, I just clicked this one, returned ore and messed up quests. I would suggest adding a choice into this dialogue that would return ore. This would give player an option to confirm he really wants to return ore bak.

https://github.com/Fawkes-dev/G1-CZ-Revision/blob/485f7a02cad28977296daed2e6bf4db73f78517c/content/Story/Missions/DIA_STT_309_Whistler.d#L223-L270

**Comment by AmProsius** [comment]
That's a very opinionated bug fix to be honest. There's nothing wrong with the quest as is.

**Comment by Fawkes-dev** [comment]
Yes, it is 💯 % 😆 I am thinking it might be 'QoL improvement 'worth adding in. Will see if we agree with the rest of the team to go for it.
